packages/flutter/lib/src/widgets/container.dart

```diff
@@ -356,6 +357,9 @@ class Container extends StatelessWidget {
   /// The transformation matrix to apply before painting the container.
   final Matrix4? transform;
 
+  /// The transform alignment
```

Comment:
nit: end with a `.`

Also: this doc needs a little more details about what this property does.